bot.py

```diff
@@ -216,23 +202,34 @@ async def get_rank(ctx, *, name: str):  # using * for get a summoner name with s
         await ctx.send(embed=create_embed(embed_data))
 
 
+# TODO: REWORK THIS WITHOUT pd
 @bot.command(
     name="last_match",
     help="Displays the information about the latest game of the summoner.",
 )
 async def get_last_match(ctx, *, name: str):
     """Sends the summoner's last match information to the bot"""
     try:
-        last_match_info = previous_match(name)
-        dfi.export(last_match_info, "df_styled.png")
-        file = discord.File("df_styled.png")
-        embed = discord.Embed()
-        embed.set_image(url="attachment://df_styled.png")
-        await ctx.send(embed=embed, file=file)
-        os.remove("df_styled.png")
+
+        # last_match_info = previous_match(name)
+        embed = discord.Embed(
+            title="last match",
```

Comment:
@jung1411 I'm silencing this command - don't like the look of the pic + generating images with same images can break the code when there are multiple requests.
We should re-work this

Reply:
Okay, but I think we don't really need this command anyways since no one is actually going to use it.

Reply:
agreed, I didn't like how we create image and the image was not legit.
Since this is expected to be not used frequently, we can put this in lower priority, but we should have this legit before making available to customers